### ISSUE TITLE
[翻訳フィードバック] 計算グラフの微積分：バックプロパゲーションを理解する

### ISSUE THREAD

**Issue by pn11** [postdcc/feedback#97]
### 記事タイトル or URL

計算グラフの微積分：バックプロパゲーションを理解する

### フィードバック内容

「はじめに」にかなり大きな間違いがあります。

> For modern neural networks, it can make training with gradient descent as much as ten million times faster, relative to a naive implementation. 

が

> 現代のニューラルネットワークでは、最急降下法という計算法を使えば単純な実装と比べて1000万倍の速さで学習ができます。

と訳されていますが、

> 最近のニューラルネットワークではバックプロパゲーション (誤差逆伝播法) を使うことで、最急降下法による学習が愚直な実装と比べて1000万倍速くなります。

のようになるべきかと思います。
つまり it は誤差逆伝播法のことで、それとナイーブな実装とを比べています。最急降下法は誤差逆伝播の場合でも、ナイーブな場合でも学習に使います。これに続く文でも最急降下法と誤差逆伝播法を取り違えています。

その他マイナーなフィードバックも挙げておきます。

- 原文でも同様ですが `Griewank（2010）` はリンク切れしていました。
- `functional programs` は `関数プログラム` と訳されていますが `関数型プログラミング` などとするのが良いかと思います。
- `Theano.` に必要ないピリオドがついています。

**Comment by Seiichi-Akiba** [comment]
フィードバックありがとうございます。
ご指摘の頂いた内容を確認して修正いたしました。
今後ともPOSTDをよろしくお願いいたします
